engine/gfx/sprite_palettes.asm

```diff
@@ -85,12 +135,23 @@ CopySpritePal::
 	rst AddNTimes
 .got_pal
 	pop de
+	push de ; push wOBPals1 palette *
 	ld bc, 1 palettes
 	call FarCopyColorWRAM
-	ld hl, wPalFlags
-	bit NO_DYN_PAL_APPLY_F, [hl]
+	ld a, [wPalFlags]
+	and NO_DYN_PAL_APPLY
+	pop hl ; pop wOBPals1 palette *
```

Comment:
The condition-checking `ld :: and` doesn't need to be inside the `push :: ... :: pop`, so IMO it should stay closest to the subsequent conditional `jr`, and then the `push :: ... :: pop` will be closer together.
```suggestion
	pop hl ; pop wOBPals1 palette *
	ld a, [wPalFlags]
	and NO_DYN_PAL_APPLY
```